java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -49,7 +49,7 @@ public static String getLinuxVersion() throws IOException {
             Process process = Runtime.getRuntime().exec("lsb_release -r");
             reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
             String line;
-            Pattern releasePattern = Pattern.compile("Release:\\s*(\\d+).*");
+            Pattern releasePattern = Pattern.compile("Release:\\s*(\\S+).*");
```

Comment:
This is problematic then because if we go with this we'll have to create a separate build for every minor release as well and the number of libraries packaged in the jar will explode.